primeqa/tableqa/preprocessors/dataset.py

```diff
@@ -0,0 +1,58 @@
+from cmath import nan
+import torch
+import pandas as pd
+from nlp import load_dataset
+import nlp
+import ast
+
+class DatasetProcessor(torch.utils.data.Dataset):
```

Comment:
Is there a reason why we do not work with HF Dataset objects?

Reply:
We are using torch Dataset Object to preprocess and convert wikisql dataset into Tensors.